var/spack/repos/builtin/packages/mariadb/package.py

```diff
@@ -63,6 +64,7 @@ class Mariadb(CMakePackage):
     depends_on("krb5")
 
     conflicts("%gcc@9.1.0:", when="@:5.5")
+    conflicts("%gcc@13:", when="@:10.8.2")
```

Comment:
This is just the last Spack version that does not build with GCC 13, not the actual MariaDB before this was fixed, correct?

Reply:
Correct. If we want to be more specific, it was actually fixed here: https://github.com/MariaDB/server/commit/75bbf645a66db797be2abd3a348dce32eb753acc

Reply:
Then let's use the correct constraint, so probably something like `@:10.5.19,10.6:10.6.12,10.8:10.8.7,10.9:10.9.5,10.10:10.10.3,10.11:10.11.2,11.0.0,11.1.0`?

Reply:
Have updated it properly. Tracking back the file, `@10.11.2` was the last conflicting version (fix was applied in `10.11.3`).